### DISCUSSION TITLE
Woocommerce

### DISCUSSION THREAD

**Discussion by philmitch** [dalenguyen/firebase-wordpress-plugin#141] [Ideas]
I think it would be helpful to add whether someone is an active user to the plugin. Or that users current subscription and/or membership.

**Comment by dalenguyen** [comment]
Hi @philmitch, can give more examples of your use cases? 

All I can think of for now is Users have their data in firebase database (firestore / realtime), then in order to know if a user is active or not is to add `lastSignIn` field to their data. 

And current subscription / membership can be synced to orders / subscriptions collection in Firestore for example.
